config/database.yml

```diff
@@ -1,31 +1,53 @@
-# This file will be overwritten by configuration manager
-#   gem install mysql2 
+# MySQL. Versions 5.1.10 and up are supported.
 #
-#   Ensure the MySQL 2 gem is defined in your Gemfile
+# Install the MySQL driver
+#   gem install mysql2
+#
+# Ensure the MySQL gem is defined in your Gemfile
 #   gem 'mysql2'
-development:
+#
+# And be sure to use new-style password hashing:
+#   http://dev.mysql.com/doc/refman/5.7/en/old-client.html
+#
+default: &default
   adapter: mysql2
+  encoding: utf8
+  pool: <%= ENV.fetch("RAILS_MAX_THREADS") { 5 } %>
+  username: root
+  password: password
+  host: localhost
+
+development:
+  <<: *default
   database: hydranorthdev
-  username: root 
-  password:
-  pool: 5
-  timeout: 5000
 
 # Warning: The database defined as "test" will be erased and
 # re-generated from your development database when you run "rake".
 # Do not set this db to the same as development or production.
 test:
-  adapter: mysql2
+  <<: *default
   database: hydranorthtest
-  username: root
-  password:
-  pool: 5
-  timeout: 5000
+  url: <%= ENV['DATABASE_URL'] %>
```

Comment:
something got mangled here? Needs a staging namespace above this. After this is fixed, ill reapprove